src/controllers/recipes_controller.js

```diff
@@ -27,6 +28,30 @@ module.exports.searchRecipes = ({ body: { query = null } }, res) => {
   });
 };
 
+module.exports.getTopRecipes = (req, res) => {
+  const numberOfRecipes = 8;
+  usersModel.countLikedRecipes((err, recipes) => {
+    if (!err && Object.keys(recipes).length >= numberOfRecipes) {
```

Comment:
So if we can't return at least 8 recipes, we just get a random 8 instead? Also should we add the ability to requests a specific amount?